indexing.sql

```diff
@@ -0,0 +1,37 @@
+CREATE USER indexed_cars_user;
+CREATE DATABASE indexed_cars OWNER indexed_cars_user;
+\c 'indexed_cars';
+
+SELECT COUNT(*) FROM car_models; 
```

Comment:
This isn't needed, also, how are you getting the tables built? And how are you importing the data 10 times?